#!/usr/bin/env python
PACKAGE = "fw_ctrl"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("T_lat", double_t, 0, "lateral-directional guidance period [s]", 5.0, 0.1, 10.0)
gen.add("T_lon", double_t, 0, "longitudinal guidance period [s]", 3.0, 0.1, 10.0)
gen.add("gamma_app_max", double_t, 0, "maximum approach flight path angle [deg]", 20.0, 1.0, 70.0)
gen.add("use_occ_as_guidance", bool_t, 0, "sum occlusion normals in guidance vector", False)

exit(gen.generate(PACKAGE, "fw_ctrl", "guidance"))
